types/remove-files-webpack-plugin/index.d.ts

```diff
@@ -0,0 +1,106 @@
+// Type definitions for remove-files-webpack-plugin 1.0
+// Project: https://github.com/Amaimersion/remove-files-webpack-plugin/blob/master/README.md
+// Definitions by: Sergey Kuznetsov <https://github.com/Amaimersion>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/**
+ * A plugin for webpack which removes files and folders before and after compilation.
+ */
+// tslint:disable-next-line:no-unnecessary-class
```

Comment:
Please remove the disable rule and fix errors

Reply:
@sheetalkamat There is no errors actually. `RemovePlugin` is a class, not a function. Unfortunately, i cannot override default tslint config. Default tslint config don't allow only constructor classes. In fact, it is incorrect. 

So, if i remove this line, then i need to either change a class to function, that is incorrect at all, or add to class some unnecessary method that doesn't part of my plugin. You can convinced of it – https://github.com/Amaimersion/remove-files-webpack-plugin/blob/master/src/plugin.js#L80

Also, you, in README, allow to add some disable rules for linter – https://github.com/DefinitelyTyped/DefinitelyTyped#lint

So, it is not against the rules, it is not against the code. That line should exists.